rules/rulelist.go

```diff
@@ -77,6 +77,7 @@ func Generate(filters ...RuleFilter) RuleList {
 		"G301": {"Poor file permissions used when creating a directory", NewMkdirPerms},
 		"G302": {"Poor file permisions used when creation file or using chmod", NewFilePerms},
 		"G303": {"Creating tempfile using a predictable path", NewBadTempFile},
+		"G304": {"Audit use of file inclusion", NewReadFile},
 
```

Comment:
See my suggestion above regarding the name of this rule.

Reply:
will do